operator/gitops/argocd/pipeline-service/openshift-pipelines/pipelines-scc.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: pipelines-scc
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+  - SETFCAP
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups:
+  - system:cluster-admins
```

Comment:
this is also an addition from the `restricted` SCC

From the godoc, this is just saying that cluster admins can use this SCC ... that is OK.

It is not saying that this SCC grants cluster-admin privs

https://github.com/openshift/api/blob/master/security/v1/types.go#L124-L127